frontend/rolecall/src/app/performance/performance-editor.component.html

```diff
@@ -274,47 +323,60 @@ <h2 class="title">Program</h2>
         </div>
         <div class="children-container">
           <div class="child-container">
-            <div *ngFor="let segment of step2Data; let i = index" class="child app-card"
-              [ngClass]="segment.uuid == this.selectedSegment.uuid && selectedIndex == i ? 'child-selected' : ''"
-              (click)="onSelectStep3Segment(segment, i)">
+            <div *ngFor="let segment of step2Data; let i = index"
+                class="child app-card"
+                [ngClass]="segment.uuid == this.selectedSegment.uuid && selectedIndex == i ? 'child-selected' : ''"
+                [ngStyle]="{'margin-left': segment.siblingId > 0 ? '2rem' : '0rem' }"
```

Comment:
Can be simplified to remove some quotes and brackets, and to limit the scope of the statement:
`[style.margin-left.rem]="segment.siblingId > 0 ? 2 : 0"`

Reply:
Neat. (Especially .rem at the end.) Thanks!